spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,35 @@
+require '/Users/ok/Projects/individual-challenges/chitter-challenge-apprenticeships/app.rb'
+require 'time'
+
+feature 'home page' do
+  scenario 'show list of peeps' do
+    visit '/'
+    expect(page).to have_content('I just bought a cabbage - 15:37')
+    expect(page).to have_content('Eggs are so weird am I right? - 15:37')
+  end
+  scenario 'add a peep' do
+    visit '/'
+    expect(page).not_to have_content('Office UK what even is that???????')
+    fill_in 'message', with: 'Office UK what even is that???????'
+    click_on('Post')
+    expect(page).to have_content("Office UK what even is that??????? - 15:37")
+  end
+  scenario 'search peeps' do
+    setup_test_database
+    visit '/'
```

Comment:
You could have made a few web helper methods to stop you having this many repeated lines e.g def add_peep